app/views/components/toast/test-bottom-right-draggable-position.html

```diff
@@ -0,0 +1,21 @@
+<div class="row">
+  <div class="twelve columns">
+    <button class="btn-secondary" type="button" id="show-toast-message1">Show Toast Message</button>
+  </div>
+</div>
+
+<script>
+  function showToastMsg(uniqueid, msg) {
+    $('body').toast({
+      title: 'Application Offline',
+      message: msg,
+      draggable: true,
```

Comment:
```suggestion
      draggable: true,
      savePosition: true,
```

Lets add save position or this example looks like its not working properly when you reload the page or click a toast again as it will go back to the old position.